src/compiler/watch.ts

```diff
@@ -829,7 +830,7 @@ export function setGetSourceFileAsHashVersioned(compilerHost: CompilerHost) {
  *
  * @internal
  */
-export function createProgramHost<T extends BuilderProgram = EmitAndSemanticDiagnosticsBuilderProgram>(system: System, createProgram: CreateProgram<T> | undefined): ProgramHost<T> {
+export function createProgramHost<T extends BuilderProgram = EmitAndSemanticDiagnosticsBuilderProgram>(system: System, createProgram: CreateProgram<T> | undefined, jsDocParsingMode: JSDocParsingMode | undefined): ProgramHost<T> {

```

Comment:
we should not override this. User can set the property

Reply:
This one has been removed.